modules/mono/glue/GodotSharp/GodotSharp/Core/Bridge/GodotSerializationInfo.cs

```diff
@@ -0,0 +1,87 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using Godot.NativeInterop;
+
+namespace Godot.Bridge;
+
+public class GodotSerializationInfo : IDisposable
```

Comment:
This class should be `sealed` or the disposable pattern should be implemented properly.